client/asset/btc/btc.go

```diff
@@ -2439,12 +2446,25 @@ func (btc *ExchangeWallet) convertCoin(coin asset.Coin) (*output, error) {
 // dust) for the change.
 func (btc *ExchangeWallet) sendWithReturn(baseTx *wire.MsgTx, addr btcutil.Address,
 	totalIn, totalOut, feeRate uint64) (*wire.MsgTx, *output, uint64, error) {
-	// Sign the transaction to get an initial size estimate and calculate whether
-	// a change output would be dust.
+
+	signedTx, change, fee, err := btc.signTxAndAddChange(baseTx, addr, totalIn, totalOut, feeRate)
+	if err == nil {
```

Comment:
Why not `if err != nil { return nil, nil, 0, err }`?  That's a bit more idiomatic and less confusing IMO

Reply:
Yeah... Was what I wrote initially but looked unnecessarily repetitive. It's surely clearer. Will do.